src/base.py

```diff
@@ -17,18 +17,27 @@
 
 
 def change_to_base(i,b):
-        if i== 0:
-            return [0]
-        else:
-            number_in_b=[]
-            while i>0:
-                q=(i//b)
-                r=i%b
-                number_in_b.append(r)
-                i=q
-            number_in_b.reverse()
-            better_format = int("".join(map(str, number_in_b)))
-            print(better_format)
-
-change_to_base(10000,13)
+    if i== 0:
+        return [0]
+    if i>0:
+        number_in_b=[]
+        while i>0:
+            q=(i//b)
+            r=i%b
+            number_in_b.append(r)
+            i=q
+        number_in_b.reverse()
+        better_format = int("".join(map(str, number_in_b)))
+        print(better_format)
+    else:
+        number_in_b=[]
+        while i:
+            i=abs(i)
```

Comment:
If you just change the sign of `i` you could reuse the code you have in the other branch. It is never a good idea to have duplicated code, such as here, as this increases the chance of errors and make it harder to change the code when you need to.